constitution.tex

```diff
@@ -520,9 +520,12 @@
 \asubsubsection{The Introductory Process}
 The Introductory Process is designed to provide an easy means for Introductory Members to meet existing House members, learn House history, demonstrate their involvement potential to House, and allow existing House members to evaluate them for Active Membership.
 \asubsubsubsection{The Evaluation Period}
-The Process will begin between the first and fourth week of each semester, then last between six (6) and ten (10) weeks, starting when the Evaluations Director initiates it.
+The Process will occur either once or twice per semester, and will last for six (6) weeks.
+The Process shall be initiated by the Evaluations Director during either the first and second week of each semester.
```

Comment:
Either-And is really weird wording-wise

Reply:
Resolving this as the vote is now open and the PR cannot be changed. This can be fixed in a non-semantic change if the amendment passes.